src/app/payment/paymentMethod/getPaymentMethodDisplayName.tsx

```diff
@@ -8,6 +8,12 @@ export default function getPaymentMethodDisplayName(
     return method => {
         const { displayName } = method.config;
 
+        if (method.id === PaymentMethodId.PaypalCommerceCredit) {
```

Comment:
Can we please update the tests for this too.

Reply:
Added test